### PR TITLE
Substitute Ship Label

### PR DESCRIPTION
**Pull request by TheForce172**
Implement #3757 Adding a new optional line to the missionfile for a briefing icon $Closeup label after $label. Currently has to be added in text editor though i have added a field in QTFred's briefing editor for when someone completes it.

### CODE REVIEW

**Comment by TheForce172** [comment]
Realised I forgot screenshots
image
image

**Comment by TheForce172** [comment]
Screenshots
image
image

**Review by EatThePath — COMMENTED** [review]
fwiw, the code for this looks straight forward and solid to me.

File: fred2/missionsave.cpp [hunk 1]
```diff
@@ -1046,6 +1046,15 @@ int CFred_mission_save::save_briefing()
 					fout_ext(" ", "%s", bi->label);
 				}
 
+				if (drop_white_space(bi->customClass)[0]) {
```
Comment (Goober5000) [inline fred2/missionsave.cpp:1049]: Add a `Mission_save_format != FSO_FORMAT_RETAIL` check here.

File: qtfred/src/mission/missionsave.cpp [hunk 2]
```diff
@@ -1071,6 +1071,16 @@ int CFred_mission_save::save_briefing()
 
 					fout_ext(" ", "%s", bi->label);
 				}
+				if (drop_white_space(bi->customClass)[0]) {
```
Comment (Goober5000) [inline qtfred/src/mission/missionsave.cpp:1074]: Add a `save_format != MissionFormat::RETAIL` check here.

**Review by Goober5000 — CHANGES_REQUESTED** [review]

**Review by Goober5000 — APPROVED** [review]
Looks good

**Pull request merged**